layouts/partials/pages/tree-categories.html

```diff
@@ -2,7 +2,7 @@
     <div class="main-inner">
         <div class="content categories">
             {{ if .Site.Params.displayListTitle }}
-                <h1 class="list-title">{{ .Title | default (.Type | title) }}</h1>
```

Comment:
Actually, since we have `.Title` here. You can customize this page's title by adding `content/categories/_index.md` with `title` front-matter.